_layouts/career.html

```diff
@@ -0,0 +1,66 @@
+<!Doctype html>
+<html id="careers">
+
+{% include head-header.html %}
+
+<section id="hero" class="light-text">
+</section>
+
+<section id="encyclopedia">
+    <div id="carsContent" class="noToc">
+
+        <button class="back" onClick="window.location.href='/careers/'"><img src="/images/careers/back_arrow.svg" alt="Back" style="vertical-align: middle; margin-right: 16px">Back to Careers</button>
```

Comment:
Why now simple `<a>` instead of `button` ? You don't need to do redirect via javascript here.

```
<a class="back" href="/careers/">
      <img src="/images/careers/back_arrow.svg" alt="Back to Careers" style="vertical-align: middle; margin-right: 16px">
      Back to Careers
</a>
```